### PR TITLE
prepare for publish

### PR DESCRIPTION
**Pull request by andreineculau**
* [x] cleaned up references to internal https://github.com/tobiipro/minlog
* [x] moved roundRect into browser-coords
* [x] consolidated throttle function
* [x] consolidated static values into cfg
* [x] added some references
* [x] expand into a proper README.md
* [x] add note about getting page zoom (guessing or web extension)
* [x] add note about window.devicePixelRatio
* [x] clarify in readme how concepts link to variable names
* [x] clarify purpose

---

Off-topic: first github.com/tobiipro review request 🎉 🎈 💬 🤗

### CODE REVIEW

**Comment by IanSavchenko** [comment]
General question: is this module supposed to be a valid easy-to-use (or _semi-easy-to-use_) NPM package as a result? Or it's just an open-sourced part of atex?

**Comment by andreineculau** [comment]
@IanSavchenko I don't follow entirely, but it's definitely not the latter :) (the latter just cannot be :D )

**Comment by andreineculau** [comment]
alright, I have checked all the checkboxes that I created, so now it's 100% up to your feedback to do more/change @IanSavchenko

File: README.md [hunk 1]
```diff
@@ -2,12 +2,125 @@
...
+It is close to impossible to get the page zoom from within the web page,
+but [priv/guess-zoom.js](priv/guess-zoom.js) is a proof of concept to guess it.
+
+The only way to get the real value is via a WebExtension
+and the [tabs.getZoom()](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/getZoom) API.
+
+If you are on OSX, that might be enough, but if you're on Windows, read on.
+
+
+## Display zoom and actual device pixel ratio
```
Comment (IanSavchenko) [inline README.md:106]: Can mention here that "Display zoom" is what is in code "screen.osZoomFactor"

**Review by IanSavchenko — APPROVED** [review]
A bit messy now, but we can have another iteration after integrating it back into main code.

**Comment by IanSavchenko** [comment]
Also would be good to mention in README that main purpose of the module is to know coordinates of browser window and client area in device pixels.

**Pull request merged**